misc/services/mobile-webui/mobile-webui-frontend/src/apps/huManager/containers/HUBulkActionsScreen.jsx

```diff
@@ -0,0 +1,98 @@
+import React, { useEffect, useState } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { useHistory, useRouteMatch } from 'react-router-dom';
+
+import { trl } from '../../../utils/translations';
+import { getHandlingUnitInfoFromGlobalState } from '../reducers';
+import { pushHeaderEntry } from '../../../actions/HeaderActions';
+import ButtonWithIndicator from '../../../components/buttons/ButtonWithIndicator';
+import BarcodeScannerComponent from '../../../components/BarcodeScannerComponent';
+import { toQRCodeDisplayable } from '../../../utils/huQRCodes';
+import * as api from '../api';
+import { toastError } from '../../../utils/toast';
+
+const HUBulkActionsScreen = () => {
+  const dispatch = useDispatch();
+  const history = useHistory();
+
+  const handlingUnitInfo = useSelector((state) => getHandlingUnitInfoFromGlobalState(state));
+
+  const [scannedHandlingUnitQRCodes, setScannedHandlingUnitQRCodes] = useState([handlingUnitInfo.qrCode.code]);
+  const [showHUScanner, setShowHUScanner] = useState(false);
+  const [showTargetScanner, setShowTargetScanner] = useState(false);
+
+  const { url } = useRouteMatch();
+  useEffect(() => {
+    if (!handlingUnitInfo) {
+      history.goBack();
+      return;
+    }
+
+    dispatch(pushHeaderEntry({ location: url, caption: trl('huManager.action.bulkActions.windowName') }));
+  }, []);
+
+  const onScannedHUResolved = ({ scannedBarcode }) => {
+    setScannedHandlingUnitQRCodes([...scannedHandlingUnitQRCodes, scannedBarcode]);
+    toggleHUScanner(false);
+  };
+
+  const removeHUQrCode = (huQRCodeToRemove) => {
+    const updatedScannedQRCodes = scannedHandlingUnitQRCodes.filter((qrCode) => qrCode !== huQRCodeToRemove);
+    setScannedHandlingUnitQRCodes(updatedScannedQRCodes);
+  };
+
+  const onTargetLocationScanned = ({ scannedBarcode }) => {
+    api
+      .moveBulkHUs({
+        huQRCodes: scannedHandlingUnitQRCodes,
+        targetQRCode: scannedBarcode,
+      })
+      .catch((axiosError) => toastError({ axiosError }))
+      .finally(() => toggleTargetScanner(false));
```

Comment:
in case move was a success, i think we shall go back to HU manager main screen or something.... atm how i am reading it here, it seems it does nothing, all scanned HU buttons are staying.... is this OK?

Reply:
yes, in the future there might be other bulk actions to be performed

Reply:
buuut, valid point, I'll show a toast notification with a successful message